feature-libs/product-configurator/rulebased/core/state/actions/configurator.action.ts

```diff
@@ -241,7 +241,7 @@ export class SetMenuParentGroup extends StateUtils.EntitySuccessAction {
   readonly type = SET_MENU_PARENT_GROUP;
 
   constructor(
-    public payload: { entityKey: string | string[]; menuParentGroup: string }
+    public payload: { entityKey: string | string[]; menuParentGroup?: string }
```

Comment:
Is `undefined` a valid menuParentGroup? Does it make the group a root-level group?

Reply:
Yes, will do. It means the group is on root level